packages/telemetry/src/index.ts

```diff
@@ -1,44 +1,27 @@
-import type { BinaryLike } from 'node:crypto';
-import { createHash, randomBytes } from 'node:crypto';
-
 import { isCI } from 'ci-info';
 import debug from 'debug';
-// @ts-ignore
-import gitUp from 'git-up';
-
-import { getAnonymousMeta } from './anonymous-meta.js';
-import { Config } from './config.js';
-import * as KEY from './keys.js';
+import { randomBytes } from 'node:crypto';
+import * as KEY from './config-keys.js';
+import { GlobalConfig } from './config.js';
 import { post } from './post.js';
-import { getRawProjectId } from './project-id.js';
-
-export interface AstroTelemetryOptions {
-	version: string;
-}
+import { getProjectInfo, ProjectInfo } from './project-info.js';
```

Comment:
Note: All changes in this file are from refactoring and cleanup and small bug fixing. Lots of little things tackled, but the meat of the change (the new project ID) lives in `project-info.js`.